workalendar/tests/test_america.py

```diff
@@ -2,7 +2,70 @@
 from datetime import date
 from workalendar.tests import GenericCalendarTest
 from workalendar.america import Colombia, Barbados
-from workalendar.america import Mexico, Chile, Panama, Paraguay
+from workalendar.america import Mexico, Chile, Panama, Paraguay, Argentina
+
+class ArgentinaTest(GenericCalendarTest):
+    cal_class = Argentina
+
+    def test_holidays_2018(self):
+        holidays = self.cal.holidays_set(2018)
+        self.assertIn(date(2018, 1, 1), holidays)       # 1. Año Nuevo
```

Comment:
those comments are valuable, but I think they're killing your flake8 compliance.
I'd suggest you to rearrange all of those on two lines:

```python
# Name of the holiday
self.assertIn(...)
```